modules/gathering-data-specific-features.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+// * cnv/cnv_users_guide/cnv-collecting-cnv-data.adoc
+// Dependency: modules/cnv-document-attributes.adoc
+
+[id="gathering-data-specific-features_{context}"]
+= Gathering data about specific features
+
+You can gather debugging information about specific features by using the
+`oc adm must-gather` CLI command with the `--image` argument. The `must-gather`
```

Comment:
I mentioned below, but there is --image and --image-stream. So maybe also mention "--image or --image-stream" here.